recipes/ncbi-cxx-toolkit-public/26/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.9)
+project(PackageTest)
+
+set(CMAKE_CXX_STANDARD 17)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+# the following is not working in the toolkit v26
+#conan_basic_setup(TARGETS)
+find_package(ncbi-cxx-toolkit-public REQUIRED)
+
+NCBI_begin_app(basic_sample)
+  NCBI_sources(basic_sample)
+  NCBI_uses_toolkit_libraries(xncbi)
+NCBI_end_app()
+
+add_executable(basic_sample2 basic_sample.cpp)
```

Comment:
Why is this required if the section above produces an exe 🤔 
Sorry I am notfamiliarr with this project and this seems odd with my experience so I am genuinely curious

Reply:
what exactly is required? I am not sure I get the question... if you're asking about `add_executable`, it's required to generate an executable file, that could be run in the `test_package` (usually, it's `test_package.exe`, but it's not mandatory)

Reply:
Both _NCBI_begin_app/NCBI_end_app_ and _add_executable_ produce an executable. I just want to be sure both do work.